.buildkite/pipeline.ml.yml

```diff
@@ -141,6 +141,18 @@
       --test_arg=--framework=torch
       rllib/...
 
+- label: ":brain: RLlib: RLlib tests on ray data"
+  conditions: ["NO_WHEELS_REQUIRED", "RAY_CI_DATA_AFFECTED"]
+  instance_size: medium
+  commands:
+    - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
+    - RLLIB_TESTING=1 ./ci/env/install-dependencies.sh
+    - ./ci/env/env_info.sh
+    - bazel test --config=ci $(./ci/run/bazel_export_options)
+      --build_tests_only
+      --test_tag_filters=ray_data,
```

Comment:
we may want to exclude ray_data tagged tests from the other test suites so they don't get run twice on master

Reply:
I see your concern. 
I can make this change so that this test suite gets run only once on master and on ci during any data or rllib changes

Reply:
+1 excluding the data tests in the other rllib test suites